packages/fx-core/scripts/download-templates-zip.js

```diff
@@ -32,18 +34,15 @@ async function step(desc, fn) {
   }
 }
 
-async function main() {
-  const rawTagList = await step(`Download tag list from ${config.tagListURL}`, async () => {
-    res = await axios.get(config.tagListURL);
-    return res.data;
-  });
-
+async function downloadLatestTemplates(rawTagList) {
   const tagList = rawTagList.toString().replace(/\r/g, "").split("\n");
-  const versionList = tagList.map((tag) => tag.replace(config.tagPrefix, ""));
+  const versionList = tagList
+    .filter((tag) => tag.startsWith(config.tagPrefix))
+    .map((tag) => tag.replace(config.tagPrefix, ""));
   const selectedVersion = semver.maxSatisfying(versionList, config.version);
   if (!selectedVersion) {
     console.error(`Failed to find a tag for the version, ${config.version}`);
-    exit(-1);
+    return;
```

Comment:
may be CD need to know download zips fail 
since fx-core need release with zips